docs/notebooks/plot_comparisons.ipynb

```diff
@@ -0,0 +1,1118 @@
+{
```

Comment:
<p>Very very nitpicky : "the response variable is on the probability scale". I would say "the quantity of interest is on the probability scale". In a logistic regression model the response variable is the number of successes in "n" trials. The probability, which is something we usually care about, is not an observed variable.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/695/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>